color/p5js/Law of simmilarity + color/index.js

```diff
@@ -0,0 +1,18 @@
+function setup() {
+  
+  createCanvas (800,800)
+  background (0);
+  
+
+}
+
+function draw() {
+  fill(65, 95, 98);
+  ellipse(200,200, 100, 100);
```

Comment:
Please make it dynamic. Don't repeat yourself. These 4 Dots can be crated in a loop. Also the location should be dependent on the canvas size.